Sources/SwiftDocC/Model/DocumentationNode.swift

```diff
@@ -421,6 +422,49 @@ public struct DocumentationNode {
             semantic.httpResponsesSectionVariants[.fallback] = HTTPResponsesSection(responses: responses)
         }
         
+        // The symbol ppossible values.
+        let symbolAllowedValues = symbol?.mixins[SymbolGraph.Symbol.AllowedValues.mixinKey] as? SymbolGraph.Symbol.AllowedValues
+        
+        if let possibleValues = markupModel.discussionTags?.possibleValues, !possibleValues.isEmpty {
+            let validator = PropertyListPossibleValuesSection.Validator(diagnosticEngine: engine)
+            guard let symbolAllowedValues else {
+                possibleValues.forEach { 
+                    engine.emit(validator.makeExtraPossibleValueProblem($0, knownPossibleValues: [], symbolName: self.name.plainText))
+                }
+                return semantic.possibleValuesSectionVariants[.fallback] = PropertyListPossibleValuesSection(possibleValues: [])
```

Comment:
question: what's the reasons for assigning an empty section here (as opposed to leaving the variants as-is)?

Reply:
Removed.